src/schema/schema.rs

```diff
@@ -104,10 +105,199 @@ impl Schema {
           Some(error) => error
         }
       },
-      (&SchemaType::Enum(_), &Query::Keys(_)) => Err(Error::validation("Cannot deeply query an enum.", NO_PRIMITIVE_HINT)),
+      (&SchemaType::Enum{..}, &Query::Keys(_)) => Err(Error::validation("Cannot deeply query an enum.", NO_PRIMITIVE_HINT)),
+      (&SchemaType::None, _) |
       (_, &Query::All) => Ok(())
     }
   }
+
+  /// Create a schema which does not run any validations.
+  pub fn none() -> Self {
```

Comment:
Im thinking it might be cleaner and easier at the end to just write these as a private-ish macro. But I don't think it would reduce enough to make you accept it.